addons/packages/pinniped/0.22.0/bundle/kbld-config.yaml

```diff
@@ -0,0 +1,15 @@
+#! This is necessary for kbld to pick up the image listed in the pinniped-concierge-config ConfigMap.
+---
+apiVersion: kbld.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.21.0
```

Comment:
Should this be `0.21.0` if this is a bump to `0.22.0`?

Reply:
This isn't for pinniped, this seems to be for `kbld` per discussion.

Reply:
This appears to be a `kbld` version number. It's `0.21.0` for all versions of the Pinniped package.